Wikipedia/UI-V5/WMFLocationSearchFetcher.h

```diff
@@ -1,16 +1,17 @@
 
 #import <Foundation/Foundation.h>
 @import CoreLocation;
+#import "Wikipedia-Swift.h"
+#import "PromiseKit.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface WMFLocationSearchFetcher : NSObject
 
-@property (nonatomic, assign) NSUInteger maximumNumberOfResults;
-
-- (AnyPromise*)fetchArticlesWithSite:(MWKSite*)site location:(CLLocation*)location;
-
-@property (nonatomic, assign, readonly) BOOL isFetching;
+- (AnyPromise*)fetchArticlesWithSite:(MWKSite*)site
+                            location:(CLLocation*)location
+                         resultLimit:(NSUInteger)resultLimit
+                         cancellable:(inout id<Cancellable> __nullable* __nullable)outCancellable;
```

Comment:
holy shit… this is what you have to write to pass a pointer with nullability turned on?